src/content/docs/errors-inbox/error-external-services.mdx

```diff
@@ -0,0 +1,51 @@
+metaDescription: "Connext errors inbox to external services such as Slack, Jira, and CodeStream"
+---
+
+Bring errors inbox closer to your teams tooling and workflows by connecting it to external services such as Slack, Jira, and CodeStream.
+
+## Connect an inbox to Slack [#slack]
+
+When connected to Slack, new and resurfaced error groups will be sent to a Slack channel within seconds of them occurring.  This enables your team to quickly identify any new errors or regressions, and resolve them quickly with direct links to the stack trace. This short video shows how it works (1:24 minutes):
+
+<Video id="HEbX0dgeGGw" type="youtube" />
+
+To connect an inbox to Slack:
+1. If your Slack workspace does not have the [New Relic app](https://newrelic.slack.com/apps/AP92KQJS3-new-relic?tab=more_info) installed, do that first.
+2. From an inbox, select the **Inbox Settings** icon (looks like a gear) in the top right corner.
+3. Toggle the Slack button to **on** if it is **off**.
+4. If no workspaces are available, click the <Icon name="fe-plus" /> plus button to enable Slack with a one click Slack authentication.
+5. Once authenticated, you will be able to select a Workspace and specific Channel to send notifications to.
+6. Click **Test** to ensure messages are being sent to the right channel.
+
+## Connect errors inbox to CodeStream
+
+To use CodeStream's Open in IDE integration with your APM stack trace errors, use environment variables to configure your APM agent with your application's [commit sha](https://git-scm.com/book/en/v2/Git-Tools-Revision-Selection) and/or your [release tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging) associated with the running version of your software.
+
+Once set up, you can jump from an error group directly to the offending code in your IDE by clicking the **Open in IDE** button.
+
+Learn more [here](docs/codestream/start-here/codestream-new-relic/#apm).
+
+## Connect an inbox to Jira [#jira]
+
+Connect errors inbox to Jira to easily create tickets for your errors, allowing for faster collaboration and resolution. Jira templates allow you to quickly create a ticket containing error details and links directly to the stack trace and APM for quick access and resolution. 
+
+We store a link to the the ticket alongside the associated error group for a period of time. If the error occurs again within that period, you can easily access associated tickets.
+
+<Callout variant="important">
+New Relic currently supports Atlassian-Jira Classic (company-managed) projects.
+</Callout>
+
+To connect an inbox to Jira:
+1. Click on the Jira integration icon on the far right side of the error group you want to connect to Jira. 
+  ![An image showing the location the jira integration button in errors inbox.](./images/jiraint.png "Jira integration button")
+  <figcaption>Clicking the Jira integration icon allows you to create a ticket based on a template, or create a template if you don't already have one.</figcaption>
```

Comment:
@bradleycamacho, I don't think you need a comma separating what are essentially to infinitive phrases: 
* To create a ticket based on a template
* To create a template if you don't already have one.